packages/admin/cms-admin/src/dam/FileForm/CropSettingsFields.tsx

```diff
@@ -36,32 +37,43 @@ export function CropSettingsFields({ disabled }: Props): JSX.Element {
-                    />
-                    <Box mt={2} pl={2}>
-                        <Typography variant="body2" paragraph>
-                            <FormattedMessage
-                                id="comet.dam.file.croppingInfoText"
-                                defaultMessage="Cropping selects the maximum visible area. Depending on the aspect ratio, the image may be cropped further on the page."
-                            />
-                        </Typography>
-                        <Typography variant="body2">
-                            <FormattedMessage
-                                id="comet.dam.file.focusPointInfoText"
-                                defaultMessage="The focus point marks the most important part of the image, which is always visible. Choose it wisely."
-                            />
-                        </Typography>
-                    </Box>
+                    <AdminComponentSection
+                        title={<FormattedMessage id="comet.dam.file.cropSettings.smartFocusPoint.title" defaultMessage="Smart focus point" />}
+                    >
+                        <FormControlLabel
+                            control={<Switch checked={focalPoint === "SMART"} onChange={handleSmartFocalPointChange} />}
+                            label={
+                                focalPoint === "SMART" ? (
+                                    <FormattedMessage id="comet.dam.file.smartFocusPoint.yes" defaultMessage="Yes" />
+                                ) : (
+                                    <FormattedMessage id="comet.dam.file.smartFocusPoint.no" defaultMessage="No" />
+                                )
+                            }
+                            style={{ marginBottom: "4px" }}
+                        />
+                        <Box>
+                            <Typography variant="caption" color="text.secondary" paragraph>
+                                <FormattedMessage
+                                    id="comet.dam.file.croppingInfoText"
+                                    defaultMessage="Cropping selects the maximum visible area. Depending on the aspect ratio, the image may be cropped further on the page."
+                                />
+                            </Typography>
+                            <Typography variant="caption" color="text.secondary" paragraph>
+                                <FormattedMessage
+                                    id="comet.dam.file.focusPointInfoText"
+                                    defaultMessage="The focus point marks the most important part of the image, which is always visible. Choose it wisely."
+                                />
+                            </Typography>
+                        </Box>
+                    </AdminComponentSection>
                 </FieldContainer>
                 {showChooseManualFocusPointButtons && (
                     <Field name="focalPoint">
```

Comment:
I think this also should have the `fullWidth` prop, otherwise it's `helperText` possibly breaks into a new line too early. 

```suggestion
                    <Field name="focalPoint" fullWidth>
```

Reply:
ac24b801b7a83a6569d1d6f28fe84dbe64c7e51b